### ISSUE TITLE
Define module-info

### ISSUE THREAD

**Issue by janbeernink** [omnifaces/cdi-ejb#6]

**Comment by janbeernink** [comment]
Depends on #7

**Comment by janbeernink** [comment]
As v0.2 now targets JDK 11 at minimum, this feature can be moved to v0.2.

**Comment by janbeernink** [comment]
Some issues with Shrinkwrap having split packages are currently preventing the library from being fully modularised. I'll postpone this for a later milestone.
